_unittest/test_02_2D_modeler.py

```diff
@@ -124,14 +124,14 @@ def test_create_regular_polygon(self):
 
     @pytest.mark.skipif(config["build_machine"] or is_ironpython, reason="Not running in ironpython")
     def test_plot(self):
-        self.aedtapp.modeler.primitives.create_regular_polygon([0, 0, 0], [0, 2, 0])
-        self.aedtapp.modeler.primitives.create_regular_polygon(
+        self.aedtapp.modeler.create_regular_polygon([0, 0, 0], [0, 2, 0])
```

Comment:
There are some lines, like this one, without an assert. Any reason why there is no assert? 
They are notified by codecov.
Maybe we can exclude all unit tests from code coverage.

Reply:
because those one are needed to the test assertion which is a couple of lines below